styles.css

```diff
@@ -0,0 +1,144 @@
+body {
+    font-family: "Roboto", sans-serif;
+    font-weight: 400;
+}
+
+.ticket-container {
+    margin: 50px 0 0 50px;
+    width: 782px;
+    height: 273px;
+    background: #EFC684;
+    border-radius: 10px;
+    display: flex;
+}
+
+.main-section {
+    width: 289px;
+}
+
+.ticket-text {
+    text-transform: uppercase;
+    font-size: 55px;
+    margin: 26px 0 0 20px;
+}
+
+.ticket-details-container {
+    display: flex;
+    justify-content: flex-start;
+}
+
+.ticket-details {
+    width: 82px;
+    height: 44px;
+    border: solid 1px black;
+    font-size: 20px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    margin-top: 36px;
+    margin-bottom: 39px;
+}
+
+.ticket-category {
+    margin-right: 30px;
+    margin-left:27px;
```

Comment:
Use `justify-content: space-around;` on the parent instead, or something similar. Don't position stuff with margin until absolutely necessary

Reply:
done